packages/world/src/modules/core/implementations/WorldRegistrationSystem.sol

```diff
@@ -151,9 +151,6 @@ contract WorldRegistrationSystem is System, IWorldErrors {
     string memory systemFunctionName,
     string memory systemFunctionArguments
   ) public returns (bytes4 worldFunctionSelector) {
-    // Require the caller to own the namespace
-    AccessControl.requireOwner(resourceSelector, _msgSender());
```

Comment:
you've removed a very important check from a public function 🙈

Reply:
OK I see🙈 I should add an internal function to refactor it, updated.